### ISSUE TITLE
.pb model file convert to  Paddlepaddle Error:UserWarning :no variable in your model, please ensure there are any variables in your model to save

### ISSUE THREAD

**Issue by Bigwig-learning** [PaddlePaddle/Paddle#33170]
PaddlePaddle  version：2.1.0
   tensorflow:1.14.0
   python:3.6.9
   OS:Linux 18.04

I randomly generated a ten levels model. When the .pb file of the generated model is converted to the paddlepaddle supported model, it prompts UserWarning. After conversion, there is also lack of  a model.pdiparams file . How can I solve this problem?

    
generated model:
import keras
from keras.layers import  Input,AveragePooling2D,GaussianNoise,Softmax,ReLU, ActivityRegularization,LeakyReLU, ReLU,Activation
from keras.models import Sequential,Model
import numpy as np
from keras.optimizers import SGD
inputs = Input(shape=(32,32,3))
x1 = AveragePooling2D(pool_size=(2, 2))(inputs)
x2 = GaussianNoise(stddev=0.3)(x1)
x3 = Softmax(axis=-1)(x2)
x4 = ReLU(max_value=None, negative_slope=0.0, threshold=0.0)(x3)
x5 = ActivityRegularization(l1=0.0, l2=0.0)(x4)
x6 = ActivityRegularization(l1=0.0, l2=0.0)(x5)
x7 = GaussianNoise(stddev=0.3)(x6)
x8 = Activation(activation='tanh')(x7)
x9 = ReLU(max_value=None, negative_slope=0.0, threshold=0.0)(x4)
x10 = Activation(activation='tanh')(x9)
model2 = Model(inputs=inputs,outputs=[x8,x10])
model2.summary()
model2.save('/home/jie/2021-5-23-20model-test/generated-model/10layer-one-two2/10layer-one-two2.h5')

Conversion command line：
x2paddle --framework=tensorflow --model='/root/pbFile/10layer-one-two2.pb' --save_dir=/root/paddleModelFile2-1-0/10layer-one-two2 --paddle_type dygraph

Prompt information：
![10layer-one-two2-Warning](https://user-images.githubusercontent.com/66941803/119836679-7fe74680-bf34-11eb-84e3-5df9ce75b769.png)

**Comment by SunAhong1993** [comment]
Can you send 10layer-one-two2.pb to sunyanfang01@baidu.com? We will confirm if there are any bugs.

**Comment by Bigwig-learning** [comment]
I sent the  10layer-one-two2.pb  file to you using  jiezhang2183@gmail.com this mailbox, please check it,thank you
